src/main/java/org/sonarlint/intellij/ui/SonarLintWalkthroughToolWindow.java

```diff
@@ -0,0 +1,482 @@
+    JPanel backButtonPanel, JPanel nextButtonPanel,
+    JPanel page, JLabel imageLabel) {
+    var gbc = new GridBagConstraints();
+
+    var centerPanel = createCenterPanel(stepLabel, label, pane, gbc);
+
+    gbc.anchor = GridBagConstraints.SOUTHWEST;
+
+    provideCommonButtonConstraints(gbc);
+
+    centerPanel.add(backButtonPanel, gbc);
+
+    gbc.anchor = GridBagConstraints.SOUTHEAST;
+
+    centerPanel.add(nextButtonPanel, gbc);
+
+    page.add(imageLabel, BorderLayout.NORTH);
+    page.add(centerPanel, BorderLayout.CENTER);
+  }
+
+  private static void createConnectWithYourTeamLayout(JLabel stepLabel, JLabel connectWithYourTeamLabel,
+    JScrollPane connectWithYourTeamPane,
+    JPanel connectWithYourTeamBackButtonPanel, JPanel connectWithYourTeamNextButtonPanel,
+    JPanel page, JLabel imageLabel) {
+    var gbc = new GridBagConstraints();
+
+    var page3CenterPanel = createCenterPanel(stepLabel, connectWithYourTeamLabel, connectWithYourTeamPane, gbc);
+
+    gbc.anchor = GridBagConstraints.SOUTHEAST;
+
+    provideCommonButtonConstraints(gbc);
+
+    page3CenterPanel.add(connectWithYourTeamNextButtonPanel, gbc);
+
+    gbc.anchor = GridBagConstraints.SOUTHWEST;
+
+    page3CenterPanel.add(connectWithYourTeamBackButtonPanel, gbc);
+
+    page.add(imageLabel, BorderLayout.NORTH);
+    page.add(page3CenterPanel, BorderLayout.CENTER);
+  }
+
+  private static void createReachOutToUsPageLayout(JLabel stepLabel, Project project, JLabel pageLabel, JScrollPane scrollPane,
+    JPanel backButtonPanel, JPanel panel, JLabel imageLabel) {
+    var gbc = new GridBagConstraints();
+
+    var page4CenterPanel = createCenterPanel(stepLabel, pageLabel, scrollPane, gbc);
+
+    var closeButton = new JButton("Close");
+    var closeButtonPanel = new JPanel(new FlowLayout(FlowLayout.RIGHT));
+    closeButtonPanel.add(closeButton);
+
+    gbc.anchor = GridBagConstraints.SOUTHWEST;
+
+    provideCommonButtonConstraints(gbc);
+
+    page4CenterPanel.add(backButtonPanel, gbc);
+
+    gbc.anchor = GridBagConstraints.SOUTHEAST;
+
+    page4CenterPanel.add(closeButtonPanel, gbc);
+
+    panel.add(imageLabel, BorderLayout.NORTH);
+    panel.add(page4CenterPanel, BorderLayout.CENTER);
+
+    closeButton.addActionListener(e -> {
+      var toolWindow = ToolWindowManager.getInstance(project).getToolWindow("Welcome to SonarQube for IDE");
+      if (toolWindow != null) {
+        toolWindow.hide(null);
+      }
+    });
+  }
+
+  private static @NotNull JEditorPane createReachOutToUsPageText(Font font, Project project) {
+    var descriptionPane = new JEditorPane(EDITOR_PANE_TYPE, "<html><body style=\"font-family: " + font.getFamily() +
+      "; font-size: " + font.getSize() + "pt; padding: 5px;\">" +
+      "You suspect any issue with " + SONARQUBE_FOR_IDE + "? Check the <a href=\"#logView\">log view</a>.<br>" +
+      "Share the verbose logs with us via <a href=\"#communityForum\">Community forum</a> in case of problem. We will be happy to help " +
```

Comment:
```suggestion
      "Share the verbose logs with us via our <a href=\"#communityForum\">Community forum</a> in case of a problem. We will be happy to help " +
```